spec/features/idv/inherited_proofing/inherited_proofing_cancel_spec.rb

```diff
@@ -0,0 +1,257 @@
+require 'rails_helper'
+
+# Simulates a user (in this case, a VA inherited proofing-authorized user)
+# coming over to login.gov from a service provider, and hitting the
+# OpenidConnect::AuthorizationController#index action.
+def send_user_from_service_provider_to_login_gov_openid_connect(user)
+  expect(user).to_not be_nil
+  # NOTE: VA user.
+  visit_idp_from_oidc_va_with_ial2
+end
+
+def complete_idv_steps_up_to_inherited_proofing_get_started_step(user, expect_accessible: false)
+  unless current_path.match?(/inherited_proofing[?|\/].*get_started/)
+    complete_idv_steps_before_phone_step(user)
+    click_link t('links.cancel')
+    click_button t('idv.cancel.actions.start_over')
+    expect(page).to have_current_path("#{idv_inherited_proofing_path}/get_started")
+  end
+  expect(page).to be_axe_clean.according_to :section508, :"best-practice" if expect_accessible
+end
+
+def complete_idv_steps_up_to_inherited_proofing_how_verifying_step(user, expect_accessible: false)
+  complete_idv_steps_up_to_inherited_proofing_get_started_step user,
+                                                               expect_accessible: expect_accessible
+  unless current_path.match?(/inherited_proofing[?|\/].*agreement/)
+    click_on t('inherited_proofing.buttons.continue')
+  end
+end
+
+def complete_idv_steps_up_to_inherited_proofing_we_are_retrieving_step(user,
+                                                                       expect_accessible: false)
+  complete_idv_steps_up_to_inherited_proofing_how_verifying_step(
+    user,
+    expect_accessible: expect_accessible,
+  )
+  unless current_path.match?(/inherited_proofing[?|\/].*verify_wait/)
+    check t('inherited_proofing.instructions.consent', app_name: APP_NAME), allow_label_click: true
+    click_on t('inherited_proofing.buttons.continue')
+  end
+end
+
+def complete_idv_steps_up_to_inherited_proofing_verify_your_info_step(user,
+                                                                      expect_accessible: false)
+  complete_idv_steps_up_to_inherited_proofing_we_are_retrieving_step(
+    user,
+    expect_accessible: expect_accessible,
+  )
+end
+
+feature 'inherited proofing cancel process', :js do
+  include InheritedProofingHelper
+  include_context 'va_user_context'
+
+  before do
+    allow(IdentityConfig.store).to receive(:va_inherited_proofing_mock_enabled).and_return true
+    send_user_from_service_provider_to_login_gov_openid_connect user
+  end
+
+  let!(:user) { user_with_2fa }
+
+  context 'when JS is enabled', :js do
```

Comment:
Is the `context` here necessary if (a) it's already enabled in the above `feature` metadata and (b) we have no specs which are run with JavaScript disabled?

```suggestion
```

Reply:
Done.